sql/moz-fx-data-shared-prod/telemetry_derived/newtab_conditional_daily_aggregates_v1/schema.yaml

```diff
@@ -0,0 +1,136 @@
+fields:
+- name: submission_date
+  type: DATE
+  mode: NULLABLE
+- name: country_code
+  type: STRING
+  mode: NULLABLE
+- name: pocket_enabled
+  type: BOOLEAN
+  mode: NULLABLE
+- name: pocket_sponsored_stories_enabled
+  type: BOOLEAN
+  mode: NULLABLE
+- name: topsites_enabled
+  type: BOOLEAN
+  mode: NULLABLE
+- name: total_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: total_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: home_default_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: home_default_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: newtab_default_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: newtab_default_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: home_nondefault_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: home_nondefault_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: newtab_nondefault_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: newtab_nondefault_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: welcome_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: welcome_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: searched_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: searched_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: sponsored_topsite_impressed_client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: sponsored_topsite_impressed_visit_count
+  type: INTEGER
+  mode: NULLABLE
+- name: client_count
+  type: INTEGER
+  mode: NULLABLE
+- name: visit_count
+  type: INTEGER
+  mode: NULLABLE
```

Comment:
These are already defined above.  That should be why this is failing in CI
```suggestion
```

Reply:
Thank you!